bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -250,6 +237,36 @@ export default defineComponent({
 						});
 					},
 				},
+				{
+					name: "Hull",
+					callback: (canvas: HTMLCanvasElement, bezier: WasmBezierInstance, options: Record<string, number>): void => {
+						const hullPoints: number[][][] = JSON.parse(bezier.hull(options.t));
+						hullPoints.forEach((iteration: number[][], iterationNumber: number) => {
+							iteration.forEach((point: number[], index) => {
+								const newPoint: Point = { x: point[0], y: point[1] };
+								drawPoint(getContextFromCanvas(canvas), newPoint, 3, COLORS.NON_INTERACTIVE.STROKE_1);
+								// avoid drawing lines for the first iteration (start, end and handle points)
+								if (iterationNumber === 0 || index === 0) {
+									return;
+								}
+								const prevPoint: Point = { x: iteration[index - 1][0], y: iteration[index - 1][1] };
+								drawLine(getContextFromCanvas(canvas), newPoint, prevPoint, COLORS.NON_INTERACTIVE.STROKE_1);
+							});
+						});
+					},
+					template: markRaw(SliderExample),
+					templateOptions: {
+						sliders: [
+							{
+								min: 0.01,
```

Comment:
Does `hull` have any weird behaviour at `t = 0` or `t = 1`? A lot of the points will simply be stacked on top of each other right? I think you can use the following instead (i.e. change to `min: 0` and `max: 1`):

```
templateOptions: { sliders: [tSliderOptions] }
```

Reply:
Nope, I was just using slider options from other examples. Thanks for pointing that out ill change that